<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-HKPJFGE0" ignored="0" name="BankingApp Test(failed)" tests="5" failures="4" timestamp="02 Apr 2020 09:50:51 GMT" time="48.99" errors="0">
  <testcase name="loginDDT" time="11.048" classname="com.inetbanking.testCases.TC_LoginTest_DDT_002"/>
  <testcase name="loginDDT" time="5.657" classname="com.inetbanking.testCases.TC_LoginTest_DDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginTest_DDT_002.loginDDT(TC_LoginTest_DDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="5.332" classname="com.inetbanking.testCases.TC_LoginTest_DDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginTest_DDT_002.loginDDT(TC_LoginTest_DDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="4.824" classname="com.inetbanking.testCases.TC_LoginTest_DDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginTest_DDT_002.loginDDT(TC_LoginTest_DDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="5.114" classname="com.inetbanking.testCases.TC_LoginTest_DDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginTest_DDT_002.loginDDT(TC_LoginTest_DDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- BankingApp Test(failed) -->
